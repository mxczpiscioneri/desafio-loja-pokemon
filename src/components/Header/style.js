import styled from 'styled-components'
import { AppBar, InputBase } from '@material-ui/core'
import { fade } from '@material-ui/core/styles'

export const AppBarStyled = styled(AppBar)`
  background-color: #378B7D;
  background: linear-gradient(to right, #639d25, #427768);
`

export const LogoStyled = styled.img`
  height: 56px;
`

export const SearchStyled = styled.div`
  position: relative;
  border-radius: 4px;
  background-color: ${() => fade('#FFF', 0.15)};
    &:hover {
    background-color: ${() => fade('#FFF', 0.25)};
  };
  width: 100%;
  margin-left: 0;
  @media (min-width: 600px) {
    margin-left: 24px;
  }
`

export const SearchIconStyled = styled.div`
  padding: 16px;
  height: 100%;
  position: absolute;
  display: flex;
  align-items: center;
  justify-content: center;
`

export const InputBaseStyled = styled(InputBase)`
  color: inherit;
  padding: 8px;
  padding-left: 56px;
  width: 100%;
`
