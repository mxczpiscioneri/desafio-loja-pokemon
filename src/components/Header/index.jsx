import React from 'react'
import SearchIcon from '@material-ui/icons/Search'
import { Toolbar } from '@material-ui/core'
import Tabs from '../Tabs'
import image from '../../assets/images/logo.png'
import { AppBarStyled, InputBaseStyled, LogoStyled, SearchIconStyled, SearchStyled } from './style'

const Header = () => (
  <AppBarStyled position="static">
    <Toolbar>
      <LogoStyled src={image} />
      <SearchStyled>
        <SearchIconStyled>
          <SearchIcon />
        </SearchIconStyled>
        <InputBaseStyled
          placeholder="Procurar..."
          inputProps={{ 'aria-label': 'search' }}
        />
      </SearchStyled>
    </Toolbar>
    <Tabs />
  </AppBarStyled>
)

export default Header
