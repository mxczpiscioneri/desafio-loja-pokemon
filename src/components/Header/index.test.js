import React from 'react'
import { render } from '@testing-library/react'
import Header from '.'

describe('Header components', () => {
  test('Render with success', () => {
    const component = render(<Header />)
    expect(component).toMatchSnapshot()
  })
})
