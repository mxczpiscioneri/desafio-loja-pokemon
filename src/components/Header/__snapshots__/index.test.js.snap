// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header components Render with success 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary sc-dlfnbm hCALJY MuiPaper-elevation4"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
        >
          <img
            class="sc-hKgILt bwMUMg"
            src="logo.png"
          />
          <div
            class="sc-eCssSg kccrPl"
          >
            <div
              class="sc-jSgupP nTBnX"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                />
              </svg>
            </div>
            <div
              class="MuiInputBase-root sc-gKsewC hQyvFH"
            >
              <input
                aria-label="search"
                class="MuiInputBase-input"
                placeholder="Procurar..."
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <div
          class="MuiTabs-root"
        >
          <div
            class="MuiTabs-scrollable"
            style="width: 99px; height: 99px; position: absolute; top: -9999px; overflow: scroll;"
          />
          <div
            class="MuiTabs-scroller MuiTabs-scrollable"
            style="margin-bottom: 0px;"
          >
            <div
              aria-label="scrollable auto tabs"
              class="MuiTabs-flexContainer"
              role="tablist"
            >
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="all"
                      class="MuiAvatar-img"
                      src="all.svg"
                    />
                  </div>
                  all
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="normal"
                      class="MuiAvatar-img"
                      src="normal.svg"
                    />
                  </div>
                  normal
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="fighting"
                      class="MuiAvatar-img"
                      src="fighting.svg"
                    />
                  </div>
                  fighting
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="flying"
                      class="MuiAvatar-img"
                      src="flying.svg"
                    />
                  </div>
                  flying
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="poison"
                      class="MuiAvatar-img"
                      src="poison.svg"
                    />
                  </div>
                  poison
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="ground"
                      class="MuiAvatar-img"
                      src="ground.svg"
                    />
                  </div>
                  ground
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="rock"
                      class="MuiAvatar-img"
                      src="rock.svg"
                    />
                  </div>
                  rock
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="bug"
                      class="MuiAvatar-img"
                      src="bug.svg"
                    />
                  </div>
                  bug
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="ghost"
                      class="MuiAvatar-img"
                      src="ghost.svg"
                    />
                  </div>
                  ghost
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="steel"
                      class="MuiAvatar-img"
                      src="steel.svg"
                    />
                  </div>
                  steel
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="fire"
                      class="MuiAvatar-img"
                      src="fire.svg"
                    />
                  </div>
                  fire
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="water"
                      class="MuiAvatar-img"
                      src="water.svg"
                    />
                  </div>
                  water
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="grass"
                      class="MuiAvatar-img"
                      src="grass.svg"
                    />
                  </div>
                  grass
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="electric"
                      class="MuiAvatar-img"
                      src="electric.svg"
                    />
                  </div>
                  electric
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="psychic"
                      class="MuiAvatar-img"
                      src="psychic.svg"
                    />
                  </div>
                  psychic
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="ice"
                      class="MuiAvatar-img"
                      src="ice.svg"
                    />
                  </div>
                  ice
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="dragon"
                      class="MuiAvatar-img"
                      src="dragon.svg"
                    />
                  </div>
                  dragon
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="dark"
                      class="MuiAvatar-img"
                      src="dark.svg"
                    />
                  </div>
                  dark
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
                role="tab"
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  <div
                    class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                  >
                    <img
                      alt="fairy"
                      class="MuiAvatar-img"
                      src="fairy.svg"
                    />
                  </div>
                  fairy
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
            <span
              class="PrivateTabIndicator-root-1 PrivateTabIndicator-colorSecondary-3 MuiTabs-indicator"
              style="left: 0px; width: 0px;"
            />
          </div>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary sc-dlfnbm hCALJY MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <img
          class="sc-hKgILt bwMUMg"
          src="logo.png"
        />
        <div
          class="sc-eCssSg kccrPl"
        >
          <div
            class="sc-jSgupP nTBnX"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
              />
            </svg>
          </div>
          <div
            class="MuiInputBase-root sc-gKsewC hQyvFH"
          >
            <input
              aria-label="search"
              class="MuiInputBase-input"
              placeholder="Procurar..."
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        class="MuiTabs-root"
      >
        <div
          class="MuiTabs-scrollable"
          style="width: 99px; height: 99px; position: absolute; top: -9999px; overflow: scroll;"
        />
        <div
          class="MuiTabs-scroller MuiTabs-scrollable"
          style="margin-bottom: 0px;"
        >
          <div
            aria-label="scrollable auto tabs"
            class="MuiTabs-flexContainer"
            role="tablist"
          >
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="all"
                    class="MuiAvatar-img"
                    src="all.svg"
                  />
                </div>
                all
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="normal"
                    class="MuiAvatar-img"
                    src="normal.svg"
                  />
                </div>
                normal
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="fighting"
                    class="MuiAvatar-img"
                    src="fighting.svg"
                  />
                </div>
                fighting
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="flying"
                    class="MuiAvatar-img"
                    src="flying.svg"
                  />
                </div>
                flying
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="poison"
                    class="MuiAvatar-img"
                    src="poison.svg"
                  />
                </div>
                poison
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="ground"
                    class="MuiAvatar-img"
                    src="ground.svg"
                  />
                </div>
                ground
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="rock"
                    class="MuiAvatar-img"
                    src="rock.svg"
                  />
                </div>
                rock
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="bug"
                    class="MuiAvatar-img"
                    src="bug.svg"
                  />
                </div>
                bug
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="ghost"
                    class="MuiAvatar-img"
                    src="ghost.svg"
                  />
                </div>
                ghost
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="steel"
                    class="MuiAvatar-img"
                    src="steel.svg"
                  />
                </div>
                steel
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="fire"
                    class="MuiAvatar-img"
                    src="fire.svg"
                  />
                </div>
                fire
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="water"
                    class="MuiAvatar-img"
                    src="water.svg"
                  />
                </div>
                water
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="grass"
                    class="MuiAvatar-img"
                    src="grass.svg"
                  />
                </div>
                grass
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="electric"
                    class="MuiAvatar-img"
                    src="electric.svg"
                  />
                </div>
                electric
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="psychic"
                    class="MuiAvatar-img"
                    src="psychic.svg"
                  />
                </div>
                psychic
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="ice"
                    class="MuiAvatar-img"
                    src="ice.svg"
                  />
                </div>
                ice
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="dragon"
                    class="MuiAvatar-img"
                    src="dragon.svg"
                  />
                </div>
                dragon
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="dark"
                    class="MuiAvatar-img"
                    src="dark.svg"
                  />
                </div>
                dark
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              aria-selected="false"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit sc-bdfBwQ ixxUhz MuiTab-labelIcon"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="MuiTab-wrapper"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle sc-gsTCUz dpZnWk"
                >
                  <img
                    alt="fairy"
                    class="MuiAvatar-img"
                    src="fairy.svg"
                  />
                </div>
                fairy
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
          <span
            class="PrivateTabIndicator-root-1 PrivateTabIndicator-colorSecondary-3 MuiTabs-indicator"
            style="left: 0px; width: 0px;"
          />
        </div>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
